server/service/logging_campaigns.go

```diff
@@ -0,0 +1,75 @@
+package service
+
+import (
+	"context"
+	"time"
+
+	"github.com/kolide/fleet/server/contexts/viewer"
+	"github.com/kolide/fleet/server/kolide"
+	"github.com/kolide/fleet/server/websocket"
+)
+
+func (mw loggingMiddleware) NewDistributedQueryCampaign(ctx context.Context, queryString string, hosts []uint, labels []uint) (*kolide.DistributedQueryCampaign, error) {
+	var (
+		loggedInUser = "unauthenticated"
+		err          error
+	)
+	vc, ok := viewer.FromContext(ctx)
```

Comment:
Prefer keeping these variables scoped just to the `if` where they are used:

```
if vc, ok := viewer.FromContext(ctx); ok {
        loggedInUser = vc.Username()
}
```

For other instances I will just comment "variable scope".